api/src/main/java/com/datatorrent/api/DAG.java

```diff
@@ -164,6 +164,8 @@
   {
     String getName();
 
+    Module getModule();
+
```

Comment:
Not related to this PR. Does it make sense to have getMeta for ports at module level? Can they not be derived from operator Meta? I don't see them used any where as well. Concrete class LogicalPlan.ModuleMeta also return null